### PR TITLE
Completed italian translation

### PR DESCRIPTION
**Pull request by GortiZ6**
Completed italian translation for v3.12.x

### CODE REVIEW

File: src/translations/sqlb_it.ts [hunk 1]
```diff
@@ -4457,32 +4458,32 @@ Se rispondi di no proveremo ad importare i dati del file SQL all&apos;interno de
...
         <location filename="../MainWindow.cpp" line="336"/>
         <source>Simplify Window Layout</source>
-        <translation type="unfinished"></translation>
+        <translation>Semplifica disposizione finestra</translation>
     </message>
     <message>
         <location filename="../MainWindow.cpp" line="337"/>
         <source>Shift+Alt+0</source>
-        <translation type="unfinished"></translation>
+        <translation>Shift+Alt+0</translation>
```
Comment (lucydodo) [inline src/translations/sqlb_it.ts:4471]: We are asking to contributors not to translate shortcuts on the [wiki](https://github.com/sqlitebrowser/sqlitebrowser/wiki/Translations#dont-translate-key-names-in-shortcuts).

There was a discussion about this in PR #0000 some time ago, but in that case,
the shortcut didn't work and we wasn't sure if that was a Qt bug, so we proceeded the merge without deletion.

If possible, delete the shortcut translation, and if the shortcut doesn't work after deleting,
please tell us so we can just merge it without deletion.

File: src/translations/sqlb_it.ts [hunk 2]
```diff
@@ -6127,152 +6128,152 @@ Tutte le tue preferenze andranno perse e verranno utilizzati i valori predefinit
...
         <source>Reload all data and update the views</source>
-        <translation type="unfinished"></translation>
+        <translation>Ricarica tutti i dati e aggiorna le viste</translation>
     </message>
     <message>
         <location filename="../RemoteDock.ui" line="414"/>
         <source>F5</source>
-        <translation type="unfinished"></translation>
+        <translation>F5</translation>
     </message>
```
Comment (lucydodo) [inline src/translations/sqlb_it.ts:6222]: Here too :wink:

File: src/translations/sqlb_it.ts [hunk 3]
```diff
@@ -6127,152 +6128,152 @@ Tutte le tue preferenze andranno perse e verranno utilizzati i valori predefinit
...
         <location filename="../RemoteDock.ui" line="411"/>
         <source>Reload all data and update the views</source>
-        <translation type="unfinished"></translation>
+        <translation>Ricarica tutti i dati e aggiorna le viste</translation>
     </message>
     <message>
         <location filename="../RemoteDock.ui" line="414"/>
         <source>F5</source>
-        <translation type="unfinished"></translation>
+        <translation>F5</translation>
```
Comment (lucydodo) [inline src/translations/sqlb_it.ts:6221]: Here too :wink:

**Review by lucydodo — CHANGES_REQUESTED** [review]
Thanks for contributing!
There are items that need some simple modifications. Please check them. :smile:

**Comment by GortiZ6** [comment]
Hi, I knew that request, but I found many other translated shortucts (with the same key of course) so I performed the same translation.
Anyway I removed all of them. Let me know if that's ok or you need something different.

**Review by lucydodo — APPROVED** [review]

**Comment by lucydodo** [comment]
Nothing needs to be modified. Thanks for correction. Now it just squash and merged. :smile:

**Pull request merged**